src/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Tests
 {
+    [Collection("CultureTestCollection")]
```

Comment:
Ok not waiting until something fails :) I noticed that test methods in different classes are manipulating CurrentCulture, therefore I added them to the same Collection so that they can't run at the same time. I wanted to avoid using RemoteInvoke where it's not 100% necessary.